sql/cat_tools.sql

```diff
@@ -346,6 +346,8 @@ SELECT __cat_tools.create_function(
   , OUT row_statement text
   , OUT when_clause text
   , OUT function_arguments text
+  , OUT function_name text
+  , OUT table_name text
```

Comment:
Could you describe your use case? I thought about adding this, but presumably if you already know the trigger OID then you already know what table it's on, no?

It should also be regclass instead of text.